cdap-ui/app/directives/log-viewer/log-viewer.less

```diff
@@ -14,30 +14,275 @@
+@timeline-buffer-color: #a6a6a6;
 
 my-log-viewer {
-  .row {
-    border: 1px solid lightgray;
-    padding: 10px 0;
-  }
-  .terminal {
-    background: @cdap-header;
-    color: white;
-    padding: 10px;
-    overflow: auto;
-    border: 1px inset #eeeeee;
-    border-radius: 4px;
-    min-height: 240px;
-    max-height: 600px;
-    pre {
-      background: transparent;
-      color: inherit;
-      border: 0;
-      padding: 0;
-      margin-bottom: 7px;
-      white-space: normal;
-      word-break: normal;
+  .log-viewer-container {
+
+    .logs-headerbar {
+      background-color: @headerbar-background-color;
+      height: 50px;
+      width: 100%;
+
+      div {
+        display: inline-block;
+      }
+      .log-stats-container {
+        width: 35%;
+        min-width: 150px;
+        height: 50px;
+        float: left;
+        color: white;
+        background-color: @headerbar-background-color;
+        display: table;
+        margin: auto;
+
+        .log-stats {
+          height: 50px;
+          min-width: 40px;
+          display: table-cell;
+          vertical-align: middle;
+          font-weight: bold;
+          top: -5px;
+
+          .stats-wrapper {
+            position: relative;
+          }
+
+        }
+        .log-stats-total {
+          min-width: 140px;
+        }
+        .log-stats-errors {
+          min-width: 90px;
+        }
+        .log-stats-warnings {
+          min-width: 115px;
+        }
+        .stat-name {
+          color: @headerbar-text-color;
+          margin-left: 10px;
+          margin-right: 4px;
+        }
+        .stat-count {
+          width: 30px;
+          height: 15px;
+          border-radius: 8px;
+          color: black;
+          display: inline-block;
+          position: relative;
+          top: 2px;
+        }
+        .stat-count-number {
+          position: relative;
+          top: -2px;
+        }
+        .total-num {
+          background-color: #dfdfdf;
+        }
+        .errors-num {
+          background-color: #fa4347;
+        }
+        .warnings-num {
+          background-color: #f4b400;
+        }
+      }
+
+      .log-actions {
+        width: 27%;
+        min-width: 410px;
+        height: 50px;
+        float: right;
+        background-color: @headerbar-background-color;
+        display: table;
+        margin: auto;
+
+        .log-action-container {
+          height: 50px;
+          display: table-cell;
+          vertical-align: middle;
+        }
+        .log-action-btn {
+          height: 25px;
+          width: 100px;
+          border: solid 1px white;
+          color: white;
+          background-color: @headerbar-background-color;
+          border-radius: 4px;
+          margin-right: 3px;
+          outline: grey;
+        }
+        .search {
+          position: relative;
+        }
+        .search .fa-search {
+          position: absolute;
+          top: 7px;
+          right: 8px;
+        }
+        .log-searchbar {
```

Comment:
checkout bootstrap styling. I think a lot of these can be eliminated with bootstrap helper classes